developer_docs/releasing.md

```diff
@@ -0,0 +1,41 @@
+# Releasing new Ansible version
+**Before checks**
+* Foreman’s required version in [register.rb](https://github.com/theforeman/foreman_ansible/blob/master/lib/foreman_ansible/register.rb) needs to be changed?
+* Version bump in [version.rb](https://github.com/theforeman/foreman_ansible/blob/master/lib/foreman_ansible/version.rb) is in separate commit
```

Comment:
```suggestion
* In case you want to release in the same PR with another commit, make sure that the version bump in [version.rb](https://github.com/theforeman/foreman_ansible/blob/master/lib/foreman_ansible/version.rb) is in a separate commit.
```